### ISSUE TITLE
LESS file Missing?

### ISSUE THREAD

**Issue by lupiter** [hmfaysal/Jekyll-HMFAYSAL-V2-Theme#1]
I've tried building the css from the LESS files with grunt, and I'm not getting the same output as the main.min.css that's in the repo. I guess you missed committing a file?

Visually, the difference is that the responsive slide out menus are not 'responsive', and are unformatted:

![screen shot 2014-01-27 at 10 51 11 am](https://f.cloud.github.com/assets/318331/2005443/cd43625a-86e4-11e3-86b2-d77accf83909.png)

Full diff of un-minified css at: http://pastebin.com/TEyuMV0f

Nice theme by the way!

**Comment by teviot** [comment]
+1 Awesome theme but there's this missing file...

**Comment by hmfaysal** [comment]
I will fix it ASAP, sorry for the delay

**Comment by smashwilson** [comment]
Yeah, I've noticed this as well. Attempts to reconstruct the missing file from the minified markup have had, uh, strange results.

From the diff, it looks like the `*.less` sources include additional work in progress that isn't reflected in the minified results, yet?
